metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_cloud.py

```diff
@@ -159,12 +154,18 @@ class DBTCloudConfig(DBTCommonConfig):
     rawCode
     compiledSql
     compiledCode
+""",
+    # Currently unsupported dbt node types:
+    # - metrics
+    # - snapshots
+    # - exposures
+}
+
+_DBT_GRAPHQL_QUERY = """
+query DatahubMetadataQuery_{type}($jobId: Int!, $runId: Int) {{
+  {type}(jobId: $jobId, runId: $runId) {{
+{fields}
```

Comment:
Do the double `{{` and `}}` get cleaned to single braces when we call `.format()`

Reply:
yup